python/onedal/common/serialization.hpp

```diff
@@ -0,0 +1,45 @@
+/*******************************************************************************
+* Copyright 2021 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#pragma once
+
+#include <pybind11/pybind11.h>
+
+#include "oneapi/dal/detail/serialization.hpp"
+#include "oneapi/dal/detail/archives.hpp"
+
+namespace oneapi::dal::python {
+
+template <typename T>
+pybind11::bytes serialize(const T& original) {
```

Comment:
Was scrolling through and noticed common mistake, there is no `inline` specifier for many header-defined functions. This may lead to duplicated symbols, when we add more translation units.